ibm/service/power/data_source_ibm_pi_datacenter.go

```diff
@@ -0,0 +1,94 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package power
+
+import (
+	"context"
+
+	"github.com/IBM-Cloud/power-go-client/clients/instance"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+const (
+	DatacenterRegion = "region"
+	DatacenterType   = "type"
+	DatacenterUrl    = "url"
+)
+
+func DataSourceIBMPIDatacenter() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIBMPIDatacenterRead,
+		Schema: map[string]*schema.Schema{
+			Arg_DatacenterZone: {
```

Comment:
If it is DC region. Why are we calling it a `zone` in Terraform?

Reply:
it is a zone — I have updated [data_source_ibm_pi_datacenter.go](https://github.com/IBM-Cloud/terraform-provider-ibm/pull/4874/files/92e3d44a4d96e3847a045d38adcba43d0af19f96#diff-139042b4df2bbc9be53090ac45feeab4c02ad0f221788a9426b8e8b421b55a6c)
` provider "ibm” {

    region    =   “lon"

    zone      =   "lon04"
  }`